.vscode/launch.json

```diff
@@ -9,7 +9,7 @@
       "type": "chrome",
       "request": "launch",
       "url": "http://dev.getsentry.net:8000",
-      "webRoot": "${workspaceRoot}/src/sentry/static/sentry/app"
```

Comment:
not sure if anyone used this, but it didn't work for me when I tried debugging from vscode, so I had to adapt it here. Let me know if I should revert this back